dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,10 +1,43 @@
 class RubricsController < ApplicationController
+  include Rails.application.routes.url_helpers
+
   def edit
     @rubric = Rubric.find_by(id: params[:id])
     @lesson = @rubric.lesson
   end
 
   def new
     @lesson = Lesson.find_by(id: params[:lessonId])
+    @rubric = Rubric.new
+  end
+
+  def create
+    @rubric = Rubric.new(rubric_params)
+    @lesson = @rubric.lesson
+    if @rubric.save
+      redirect_to(edit_rubric_path(@rubric.id))
+      return @rubric.id
+    else
+      render :new
+    end
+  end
+
+  def update
```

Comment:
Note that update and edit are still WIP.

Reply:
Would it mess you up a lot to remove `update` in this PR? If so, can you add a comment like `TODO(kaitie): fix and test update action`

Reply:
I think it is SO close to working!  I actually got it working for a brief second but then broke something else so I undid it all.  In an effort to focus on just getting this PR merged I will leave the TODO comment as recommended.